.storybook/preview.js

```diff
@@ -1,4 +1,63 @@
 import {addons} from '@storybook/addons'
+import {ThemeProvider} from '../src/ThemeProvider'
+import {themeGet} from '../src'
+import {createGlobalStyle} from 'styled-components'
+
+// set global theme styles for each story
+const GlobalStyle = createGlobalStyle`
+  body {
+    background-color: ${themeGet('colors.bg.primary')};
+    color: ${themeGet('colors.text.primary')}
+  }
+`
+export const globalTypes = {
+  colorMode: {
+    name: 'Color mode',
+    description: 'Color mode (day, night, auto)',
+    defaultValue: 'day',
+    toolbar: {
+      icon: 'paintbrush',
```

Comment:
Does storybook have a black and white circle icon something like this: 🌗?

Reply:
Looks like they don't 😞 I

 think a `paintbrush` should be fine 👍

Reply:
I'm sure there's a way to pass custom icons to the toolbar, but I couldn't find much after a quick search. Someday!